rest-api-spec/src/main/resources/rest-api-spec/test/cluster.state/20_filtering.yml

```diff
@@ -155,10 +155,6 @@ setup:
 
 ---
 "Filtering the cluster state returns cluster_uuid at the top level regardless of metric filters":
-  - skip:
-      version:  " - 6.3.99"
-      reason:   "cluster state including cluster_uuid at the top level is new in v6.4.0 and higher"
-      features: allowed_warnings
```

Comment:
I think for this one, we should just remove the `version: " - 6.3.99"` line, but leave the `features` line in place.